docker-entrypoint-prod.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#echo "Running migrations (Database & static files to S3 bucket) ..."
+echo "Bucket: ${AWS_S3_BUCKET}"
+echo "Bucket User: ${AWS_S3_USER}"
+echo "Bucket ID: ${AWS_S3_KEYID}"...
+
+
+function send_message {
+    curl -s \
+      --form-string "token=avu1jex2ixynqx3waz92hwrjqdezrt" \
+      --form-string "user=ufuvXYqcU6fnJbRCh7Uc9qktcbARuv" \
+      --form-string "message=$1" \
+      https://api.pushover.net/1/messages.json
```

Comment:
what is the Pushover API do?

Reply:
Ah, darn yes let me get rid of it. I was trying to test a Heroku Procfile and I could not determine if it was running or not, hence the pushover. This function needs to be deleted.